js/app.js

```diff
@@ -0,0 +1,429 @@
+                     {header: "Price", width: 140},
+                     {header: "Discount", width: 120},
+                     {header: "Subtotal", width: 120},
+                     {header: "Delete", width: 120}],
+    listHeaders02 = [{header: "Item", width: 200},
+                     {header: "Price", width: 80}];;
+
+const pageLoad = () => {
+    
+    const $activeList = $(`<div id="activeList"></div>`).appendTo("body"),
+          $tblList = $(`<table id="tblList"></table>`).appendTo($activeList),
+          $rowList = $(`<tr></tr>`).appendTo($tblList);
+    
+    listHeaders01.forEach(function(d) {
+        $(`<th>${d.header}</th>`).appendTo($rowList);
+        $(`<col width="${d.width}">`).appendTo($tblList);
+    });
+
+    const $itemsList = $(`<div id="addItems">Add multiple items <input id="addItems_exit" type="button" value="X"/></div>`).appendTo("body"),
+          $tblItems = $(`<table id="tblItems"></table>`).appendTo($itemsList),
+          $rowHeader = $(`<tr></tr>`).appendTo($tblItems),
+          $rowItems = $(`<tr></tr>`).appendTo($tblItems);
+    
+    listHeaders02.forEach(function(d) {
+        $(`<th>${d.header}</th>`).appendTo($rowHeader);
+        const $c = $(`<td class="multiple" contenteditable="true"></td>`).appendTo($rowItems);
+        $c[0].addEventListener("blur", handlerItems);
+        $(`<col width="${d.width}">`).appendTo($tblItems);
+    });
+    
+    if (budget !== null) {
+        $("#myBudget")[0].value = "$" + Number(budget).toFixed(2);
+    }
+    
+    function handlerItems() {
+
+        const tbl = this.parentElement.parentElement
+        let rowCount = tbl.rows.length;
+        const row = $(this).closest("tr");
+
+        let rVal = "";
+        row.find("td").each(function(i) {
+            rVal += this.innerText;
+            if (i == 0 && this.innerHTML.length > 0) {
+                this.innerHTML = this.innerHTML.toUpperCase()
+            } else if (this.innerHTML.length > 0) {
+                this.innerHTML = "$" + Number(this.innerHTML).toFixed(2);
+            }
+        });
+
+        let lVal = "";
+        $(tbl).find("tr").last().find("td").each(function(d) {
+            lVal += this.innerText;
+        });
+
+        if (rowCount <= 2 && rVal.length === 0) {
+            console.log("Just Right");
+        } else if (rVal.length > 0 && lVal.length > 0) {
+            const $r = $(`<tr></tr>`).appendTo(tbl);
+            listHeaders02.forEach(function(d) {
+                const $c = $(`<td class="multiple" contenteditable="true"></td>`).appendTo($r);
+                $c[0].addEventListener("blur", handlerItems);
+            });
+            adjHeight($("#addItems"), 44);
+
+        } else if (rVal.length === 0) {
+            row.remove()
+            adjHeight($("#addItems"), -44);
+        }
+    }
+    
+    const adjHeight = ($obj, val) => {
+        $obj.css("height",`${$obj.height() + val}px`);
+    }
+    
+    $(document).ready(function() {

```

Comment:
Yes, good job. The pageload function can be incorporated here as you already have worked with it.